### ISSUE TITLE
Does this repo have published papers that can be studied?

### ISSUE THREAD

**Issue by zhenqinchen** [muffin-rice/FECG-extraction#1]
I recently found your open source code while studying the field of fetal ecg. Thanks for your contribution!
When I wanted to try to run the code, I found that I needed to set the ecg data. Does this repo have published papers that can be studied?

**Comment by stwars7** [comment]
Hi! Have the same question about the data. Is there a link to the database or is it private? Thx!

**Comment by zhenqinchen** [comment]
> Hi! Have the same question about the data. Is there a link to the database or is it private? Thx!
Are you also trying to do an experiment to extract the fecg signal? I would like to ask if you can achieve the performance described in recent deep learning papers. At present, I haven't found any experimental code that I can refer to. If I implement it myself, I feel that it will not be as good as the results of existing articles.

**Comment by stwars7** [comment]
Yes, I'm also trying specifically with FECG) This is the first deep learning I'm trying to figure out, but not super successful)). About the data: there in the file load_data.py the library is connected wfdb.io (signal database). Try to read about it, links below. It may be possible to run) Have you already figured out the rest of the project? 
Links: https://wfdb.io/
https://physionet.org/content/wfdb-python/4.1.0/
https://wfdb.readthedocs.io/en/latest/wfdb.html

**Comment by zhenqinchen** [comment]
I think the data is preprocessed, so it might not work without knowing how it is preprocessed. In 'load_data.py',  code to load data:
        self.fecg_dataset = load_data(f'Data/preprocessed_data/fecg_signal')
        self.mecg_dataset = load_data(f'Data/preprocessed_data/mecg_signal')

I'm recently looking for a reference deep learning code on extracting fecg signal. If you find it, please contact me: zhenqinchen@foxmail.com. Of course, if I find it, I will share it with you.

**Comment by stwars7** [comment]
I still hope that the author will help us) I'll definitely share it with you if I find something suitable. My contact email: dautova.karina@icloud.com
Good luck!
